src/Container/Exceptions/BindingResolutionException.php

```diff
@@ -3,8 +3,9 @@
 namespace Give\Container\Exceptions;
 
 use Exception;
+use Give\Framework\Exceptions\Contracts\LoggableException;
 use Give\Framework\Exceptions\Traits\Loggable;
 
-class BindingResolutionException extends Exception {
+class BindingResolutionException extends Exception implements LoggableException {
 	use Loggable;
```

Comment:
Since we are using the `LoggableException` interface as the single type check, does the `Loggable` trait now represent added complexity in that each class is using a trait to implement an interface? Would this be better with `inheritance`, such as a base or abstract class?

Reply:
Please see my note, [here](https://github.com/impress-org/givewp/pull/5837#discussion_r634511688), as to why inheritance won't work.